fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -574,6 +574,50 @@ class ApplyMetadataMutationsImpl {
 		    .detail("LogRangeEnd", logRangeEnd);
 	}
 
+	void checkSetConstructKeys(MutationRef m) {
+		if (!SERVER_KNOBS->GENERATE_DATA_ENABLED) {
+			return;
+		}
+		if (!m.param1.startsWith(globalKeysPrefix)) {
+			return;
+		}
+		if (!toCommit) {
+			return;
+		}
+
+		if (m.param1.startsWith(constructDataKey)) {
+			// std::string s = "\xcf\xdf";
+			// Value v = encodeConstructValue(s, 100, 0x2ff, 123);
+			// TraceEvent(SevDebug,"ConstructDataDebug").detail("V", v);
+
+			uint64_t valSize, keyCount, seed;
+			Standalone<StringRef> prefix;
```

Comment:
do we need to check the prefix is empty? i am thinking some safety check before inserting data.